Eatery/Eat Now/Detail View Controller/EateriesGridViewController.swift

```diff
@@ -156,9 +156,22 @@ class EateriesGridViewController: UIViewController, MenuButtonsDelegate {
         loadData(false, completion: nil)
     }
     
+    func findEmojis(searchBarText: String) -> String{
+        let emojiDictionary: [String: String] = ["🍔": "burger", "🍟": "fries", "🍕": "pizza", "🐔": "chicken", "🌮": "taco", "🌯": "burrito", "🍳" : "egg", "🍚" : "rice", "🍝" :  "spaghetti", "💩": "nasties"]
```

Comment:
It's probably better to put the emoji conversions into a separate json file (similar to nicknames.json) so we can add more in the future.